### PR TITLE
Update contributors and developper list

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: src/site/xdoc/thanks.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
           <td><p>We are an Open Source Software Company.</p>
               <p>Our mission is to bring digital independency to our customers.</p>
               <p>We are focused on large organizations both from private and public sector. We have set ourselves the goal of helping Governments to achieve their digital sovereignty.</p>
-              <p>Linagora employs 4 James commiters and 3 PMC members. We can offer professional services related to James: support, feature development or project integration.</p>
+              <p>Linagora employs 5 James commiters and 3 PMC members. We can offer professional services related to James: support, feature development or project integration.</p>
```
Comment (quynhn) [inline src/site/xdoc/thanks.xml]: s/commiters/committers?

File: pom.xml [hunk 2]
```diff
@@ -276,34 +274,45 @@
         <developer>
             <id>matthieu</id>
             <name>Matthieu Baechler</name>
-            <email>matthieu@apache.org</email>
```
Comment (mbaechler) [inline pom.xml]: please don't change that on my behalf. I'm not part of Linagora and I do contribute with Baechler Craftsmanship organization.

Comment (chibenwa) [inline pom.xml]: Ok

Comment (chibenwa) [inline pom.xml]: You've been removed from the list.

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**